servicemesh-install/topics/install.adoc

```diff
@@ -239,7 +239,7 @@ $ oc get pods -n istio-operator
 . You can access the logs from the `istio-operator` pod with the following command, replacing `<pod name>` with the name of the pod you discovered.
```

Comment:
Line 231 until 239 also need to change for the pod name update, take a look at the update in PR 12589